go.mod

```diff
@@ -429,7 +430,7 @@ replace (
 	golang.org/x/net => golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e
 	golang.org/x/oauth2 => golang.org/x/oauth2 v0.0.0-20191202225959-858c2ad4c8b6
 	golang.org/x/sync => golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
-	golang.org/x/sys => golang.org/x/sys v0.0.0-20200201011859-915c9c3d4ccf // pinned to release-branch.go1.14-std
+	golang.org/x/sys => golang.org/x/sys v0.0.0-20200622214017-ed371f2e16b4 // pinned to release-branch.go1.14-std
```

Comment:
revert to `release-branch.go1.14-std` (915c9c3d4ccf) if possible

Reply:
I needed to upgrade golang.org/x/sys because github.com/cilium/ebpf needs the definition for `unix.BPF_F_RDONLY_PROG`